### PR TITLE
WL-4028 Sorted Links in alphabetical order.

### PR DESCRIPTION
**Pull request by ouit0408**
Added new 'QuickLinkComparator.java' class to sort the quick Links.

### CODE REVIEW

File: portal-impl/impl/src/java/org/sakaiproject/portal/charon/SkinnableCharonPortal.java [hunk 1]
```diff
@@ -23,18 +23,7 @@
 import java.io.IOException;
 import java.io.PrintWriter;
 import java.net.URLEncoder;
-import java.util.ArrayList;
```
Comment (buckett) [inline portal-impl/impl/src/java/org/sakaiproject/portal/charon/SkinnableCharonPortal.java:26]: Can you not cleanup imports as this shouldn't be needed for this change and makes merging with Sakai tricky.

File: portal-impl/impl/src/java/org/sakaiproject/portal/charon/QuickLinkComparator.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+/**
+ * Created by neelam on 24/07/2015.
+ * Comparator to Sort the quick links
+ */
+public class QuickLinkComparator implements Comparator {
+
+    public int compare(Object object1, Object object2) {
+        Map<String, String> quickLink1 = (Map<String, String>) object1;
+        Map<String, String> quickLink2 = (Map<String, String>) object2;
+        return quickLink1.get("name").compareTo(quickLink2.get("name"));
```
Comment (buckett) [inline portal-impl/impl/src/java/org/sakaiproject/portal/charon/QuickLinkComparator.java]: Will NPE if quicklink1 doesn't have a name which from the building of the list looks to be possible.

**Comment by buckett** [comment]
Actually no NPE.

**Comment by buckett** [comment]
Why not just fix the config so it's alphabetical?

**Comment by ouit0408** [comment]
have changed the 'placeholder.properties' in sakai-prod to sort the quick links in alphabetical order, pushed my changes in the repository wl-docker. Deleted my changes and pushed it as well.

**Comment by buckett** [comment]
Closed as the `placeholder.properties` fix is done.

**Pull request closed without merging**